### PR TITLE
[10.0][FIX] add_domain was not being evaluated (#554)

### PR DESCRIPTION
**Pull request by ivantodorovich**
This fixes https://github.com/OCA/website/issues/554

At first, I tried to use `pyeval`, but then realized it's not available on website, only on backend.

### CODE REVIEW

File: website_field_autocomplete/controllers/main.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 from odoo import http
 from odoo.http import request
 from odoo.addons.website.controllers.main import Website
+import logging
```
Comment (pedrobaeza) [inline website_field_autocomplete/controllers/main.py]: Why the changes in this file?

Comment (ivantodorovich) [reply website_field_autocomplete/controllers/main.py]: Sorry, that shouldn't be there 👍 
Fixed

**Review by simahawk — APPROVED** [review]
LG
